app/controllers/issue_tta_reports_controller.rb

```diff
@@ -1,24 +1,49 @@
 class IssueTtaReportsController < ApplicationController
   def create
-    @issue_tta_report = IssueTtaReport.new(
-      issue_id: issue_id,
-      tta_report_display_id: tta_report_display_id,
-      tta_report_id: activity_data[:id]
-    )
-
-    if @issue_tta_report.save
-      redirect_to complaint_path(issue_id)
+    if activity_data[:success]
```

Comment:
[idle thought] this controller is getting so unwieldy that it's starting to feel like maybe we should have an ApiResponse object but also that's probably an overoptimization for right now